eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -298,6 +298,10 @@ jobs:
         timeoutInMinutes: ${{ variables.runTestsTimeout }}
 
   - ${{ else }}:
+    - ${{ if eq(variables['Agent.Os'], 'Darwin') }}:
```

Comment:
I think we will need a dependency installation step so from that point of view it's not temporary, but it might look a bit different that this. keeping it as is now.

Reply:
OK. FWIW we already have such a step for Windows: https://github.com/dotnet/sdk/blob/cf30ac26abdbfe9fd8bacec2c3d82094cf0d60a7/eng/pipelines/templates/jobs/vmr-build.yml#L275-L281